src/backend/preprocessor/cfgGenerator/FunctionCall.js

```diff
@@ -0,0 +1,47 @@
+define([
+    '../Cfg'
+], function (Cfg) {
+    var cfgGenerator;
+
+    function FunctionCall (paramNode, options) {
+	var functionName = paramNode.name;
+	var isVariadic = options && options.prototypes[functionName].isVariadic;
+	var parameters = paramNode.parameters;
+
+	var resolveInstr = new Cfg({
+	    type: 'RESOLVE',
+	    param: functionName
+	});
+	var result;
+
+	if(parameters && parameters.length > 0) {
```

Comment:
same here (best use a regex to change all `if(` into `if (` and `for(` into `for (`